src/index.ts

```diff
@@ -24,6 +30,10 @@ import * as _parser from "./parser";
 export { _parser as parser };
 import { URL } from "url";
 import * as path from "path";
+import { Return } from "./parser/Statement";
+import pSettle from "p-settle";
+import crypto from "crypto";
```

Comment:
is this used?

Reply:
Heck, nope!  I was originally using `"brs" + md5(someComponentLibraryUri)` as the prefix for the temp directory, but I switched towards `sanitize-filename` since that more accurately describes what we want.